x11-misc/rofi-wayland/rofi-wayland-1.7.5.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S=${WORKDIR}/rofi-${PV}+wayland1
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun test +windowmode"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/glib:2
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango[X]
+	x11-libs/startup-notification
```

Comment:
meson.build says:
```
if xcb_enabled
    deps += [
        libgwater.get_variable('libgwater_xcb'),
        dependency('xcb'),
        dependency('xcb-aux'),
        dependency('xcb-xkb'),
        dependency('xkbcommon-x11'),
        dependency('xcb-ewmh'),
        dependency('xcb-icccm'),
        dependency('xcb-randr'),
        dependency('xcb-cursor'),
        dependency('xcb-xinerama'),
        dependency('cairo-xcb'),
        dependency('libstartup-notification-1.0'),
    ]
endif
```
So these deps are only pulled when -Dxcb=enabled, which we hardwire to disabled later. So, drop all of them.